docs/deployment.md

```diff
@@ -33,7 +33,8 @@ Usage: uvicorn [OPTIONS] APP
 Options:
   --host TEXT                     Bind socket to this host.  [default:
                                   127.0.0.1]
-  --port INTEGER                  Bind socket to this port.  [default: 8000]
+  --port INTEGER                  Bind socket to this port. If 0, a random
```

Comment:
Let's use the following sentence instead, please:

"Bind socket to this port. If 0, an available port will be picked."

Reply:
@Kludex I agree with your suggestion and have made the necessary changes. Thank you for your assistance.